doc/man/zcashd.1

```diff
@@ -364,16 +360,24 @@ Prepend debug output with timestamp (default: 1)
 .HP
 \fB\-minrelaytxfee=\fR<amt>
 .IP
-Fees (in ZEC/kB) smaller than this are considered zero fee for relaying
-(default: 0.000001)
+Fees (in ZEC/kB) smaller than this are considered zero fee for relaying,
+mining and transaction creation (default: 0.000001)
+.HP
+\fB\-maxtxfee=\fR<amt>
+.IP
+Maximum total fees (in ZEC) to use in a single wallet transaction or raw
+transaction; setting this too low may abort large transactions (default:
+0.10)
 .HP
 \fB\-printtoconsole\fR
 .IP
 Send trace/debug info to console instead of debug.log file
+.PP
+Chain selection options:
 .HP
 \fB\-testnet\fR
 .IP
-Use the test network
+Use the test chain
```

Comment:
Huh, when did upstream change this? "Test network" is the term that the spec uses (and in fact it is a separate network as well as a separate chain). Oh well, I guess it's not important.